0_tutorial_cgraph/app.cpp

```diff
@@ -3,6 +3,15 @@
 #include <numeric>
 #include <taichi/cpp/taichi.hpp>
 
+namespace {
+void ti_check_error(const std::string& msg) {
```

Comment:
I wonder if something like `check_taichi_error` could be better? It somehow sounds like a part of C-API.

Reply:
SG!